### DISCUSSION TITLE
Load full article immediately: please add setting

### DISCUSSION THREAD

**Discussion by rudolfmu** [spacecowboy/Feeder#151] [Ideas]
Please add a setting to load the full article immediately from the overview. Thank you for giving us your wonderful app.

**Comment by spacecowboy** [comment]
> Please add a setting to load the full article immediately from the overview. Thank you for giving us your wonderful app.

Not sure what you mean with "overview".

But you can enable "Fetch full article by default" if you open the feed's settings.

**Comment by rudolfmu** [comment]
Thank you. I simply did not find this setting.

Am 6. Dezember 2023 08:32:52 UTC schrieb Jonas Kalderstam ***@***.***>:
>> Please add a setting to load the full article immediately from the overview. Thank you for giving us your wonderful app.
>
>Not sure what you mean with "overview".
>
>But you can enable "Fetch full article by default" if you open the feed's settings.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/spacecowboy/Feeder/discussions/151#discussioncomment-7773608
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.
